benchmark/benchmark.toml

```diff
@@ -0,0 +1,19 @@
+# TOML document for benchmark tests
+
+title = "TOML Example"
+atom = 6.626e-34
+cash = true
+chars = ["a", "b", "?", "!", "`", "-", "  d", "\t", "\\", "\"" ]
```

Comment:
I think it's better to use here different words of different types, like:

```toml
words = [ "all", 'strings', """are the same""", '''type'''] 
```

I think we don't need to have `tomlData :: TOML` specified as EDSL. We can parse file to get `TOML` object, no need to specify TOML twice in multiple places. Because we can't specify everything using pretty-printer, but we really want to parse different types.